### PR TITLE
chore: add Node.js 10

### PR DESCRIPTION
**Pull request by DanielRuf**

### CODE REVIEW

File: .circleci/config.yml [hunk 1]
```diff
@@ -34,3 +31,14 @@ jobs:
       - run:
           name: Lint and Test packages
           command: yarn test
+jobs:
+  build:
+    docker:
+      - image: circleci/node:8
```
Comment (raphamorim) [inline .circleci/config.yml]: IDK if it's necessary to run with node10 and node8. 🤔

Comment (DanielRuf) [reply .circleci/config.yml]: Node 8 is LTS and 10 current stable release. They are very different and both should be checked.

Comment (fabiomcosta) [reply .circleci/config.yml]: is this going to slow down the test build significantly?
if not, sure why not?

Comment (DanielRuf) [reply .circleci/config.yml]: It should not as CircleCI runs them in parallel.

Comment (DanielRuf) [reply .circleci/config.yml]: Would be great to have CircleCI enabled for PRs.

**Pull request merged**